parsers/ZA.py

```diff
@@ -0,0 +1,966 @@
+from __future__ import annotations
+
```

Comment:
```suggestion
```
Not sure why but this breaks pytest (and the CI) so let's remove it for now.

Reply:
This is only needed if you're going to reference a type that you define in this file